### PR TITLE
Move iopendocumentstracker

### PR DESCRIPTION
**Pull request by smoothdeveloper**
First step for #1410

### CODE REVIEW

File: src/FSharpVSPowerTools.Logic/ProjectSystem/OpenDocumentsTracker.fs [hunk 1]
```diff
@@ -15,23 +16,24 @@ type OpenDocument =
...
       LastChangeTime: DateTime
       ViewCount: int }
-    member x.Text = lazy (x.Snapshot.GetText())
     static member Create document snapshot encoding lastChangeTime = 
         { Document = document
           Snapshot = snapshot
           Encoding = encoding
           LastChangeTime = lastChangeTime
           ViewCount = 1 }
+    member private x.text = lazy (x.Snapshot.GetText())
```
Comment (vasily-kirichenko) [inline src/FSharpVSPowerTools.Logic/ProjectSystem/OpenDocumentsTracker.fs]: Why this extra property? Why not just use x.Text?

Comment (smoothdeveloper) [inline src/FSharpVSPowerTools.Logic/ProjectSystem/OpenDocumentsTracker.fs]: Can't use Text without changing call sites due to how interfaces are explicitly implemented in F#, I'd need to change call sites and lose access to other properties.

**Comment by vasily-kirichenko** [comment]
It failed to compiled.

**Comment by smoothdeveloper** [comment]
@vasily-kirichenko sorry I forgot the added file, it seems to pass now.

**Comment by vasily-kirichenko** [comment]
Great.

**Pull request merged**